leafsmart-front-end/src/components/Dashboard.js

```diff
@@ -32,7 +32,7 @@ const Dashboard = () => {
       <section className="bg-Isabelline py-5 min-h-screen">
         <div className="flex p-8 justify-between">
           <div className="w-[58%]">
-            <div className="flex justify-between">
+            <div className="flex justify-around">
```

Comment:
Can this change actually be reverted? 
Restyling of the spacing was approached slightly differently on the connect-fav-to-search branch, so this'll cause a conflict.

Reply:
Reverted